src/error_functions.py

```diff
@@ -5,18 +5,34 @@
 def mean_euclidean_error(predicted, target):
     """
     Computes the Mean Euclidean Error between
-    the targ vector and the output pred by the net
+    the targ vector and the output pred by the net over all patterns
 
     :param predicted: ndarray of shape (n, m) – Predictions for the n examples
     :param target: ndarray of shape (n, m) – Ground truth w_vals for each of n examples
     :return: loss in term of mee (Mean Euclidean Error)
     """
+
     if predicted.shape != target.shape:
-        raise Exception(f"Mismatching shapes in MSE: predictions shape: "
+        raise Exception(f"Mismatching shapes in MEE: predictions shape: "
```

Comment:
Cambiamolo in `AttributeError`,
poi occhio al corrispondente unit test

Reply:
Sistemato

Reply:
Occhio che mi sa tanto che non mi funzionano i test su mac. Quando testo le singole unit test l IDE mi da errore e mi dice di importare i file con 'src.". Una volta fatto, su PyCharm mi funzionano, invece i test su git danno errore proprio per il fatto che è  stato usato "src."

Reply:
Devi togliere "src." dagli import, su pycharm devi impostare `src` come "source directory" o qualcosa di simile